typesafety/test_issue_49.yml

```
@@ -7,3 +7,51 @@
     logger.debug(123.3)
     logger.trace(1)
     logger.error(.3)
+- case: variables
```

Hmm...name of this case is little off. Point of this test is to make it obvious we can take `NameExpr` and if this correct, plugin wont capture it. That being said, I'd propose something like

```suggestion
- case: variables_str_no_args
```